config/locales/views/schools/schools.yml

```diff
@@ -326,6 +326,17 @@ en:
       audit:
         message_html: You have completed <strong>%{completed_activities_count}/%{total_activities_count}</strong> of the activities and <strong>%{completed_actions_count}/%{total_actions_count}</strong> of the actions from your recent energy audit
         summary_html: Complete the others to score <strong>%{remaining_points}</strong> points and <strong>%{bonus_points}</strong> bonus points for completing all audit tasks
+      completed_programme:
+        button: View programme
```

Comment:
Maybe just "View"? Could then reuse the translation we're using on the programme index page, so the buttons are consistently labelled.

Reply:
Yes no probs! Makes sense